linkerd/main/src/main/scala/io/buoyant/linkerd/Main.scala

```diff
@@ -64,10 +64,16 @@ object Main extends App {
   private def initAdmin(
     config: Linker.LinkerConfig,
     linker: Linker
-  ): Closable with Awaitable[Unit] = {
+  ): Seq[Closable with Awaitable[Unit]] = {
     val server = linker.admin.serve(this, LinkerdAdmin(config, linker))
     log.info(s"serving ${linker.admin.scheme} admin on ${server.boundAddress}")
-    server
+
+    config.admin.flatMap(_.httpIdentifierPort) match {
+      case Some(p) =>
+        val idServer = linker.admin.serveHandler(new InetSocketAddress(p.port), LinkerdAdmin.identifierHandler(config, linker))
```

Comment:
Hmm, what about serving it at a particular URL, consuming the expected path, and then using the rest of the path for identification? From a user's perspective this would feel similar to the delegation endpoints.

My primary concern is when we need this info (via admin ui or curl) our users will need to redeploy with a new port open in their k8s/ELB/whatever, and/or we update linkerd-examples to expose this new port. I guess this comes down to internal complexity vs. user complexity?